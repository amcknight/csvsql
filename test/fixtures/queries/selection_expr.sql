SELECT "Hello " ++ first_name AS hello_name, age + 5 * 2 as big_age
FROM test/fixtures/tables/people
WHERE last_name = "McKnight"
