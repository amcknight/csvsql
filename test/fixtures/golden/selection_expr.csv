hello_name, big_age
Hello Andrew, 45
Hello July, 10
